### DISCUSSION TITLE
Is there any command line(cmd) to check how many unresolved conversations are in any dedicated git repo/branch?

### DISCUSSION THREAD

**Discussion by Vibhu-Kansal** [community/community#20203] [Other Feature Feedback, Questions, & Ideas]
Is there any command line(cmd) to check how many unresolved conversations are in any dedicated git repo/branch?
Like there are multiple PRs were already merged and in some of the PRs some open conversations. We need to check how many conversations happened and how many are not marked as resolved.

**Comment by diegoleondev** [comment]
Hi @Vibhu-Kansal , Unfortunately not, when interacting with the terminal we are working with Git, Git is indifferent to Github.

You can get which commands are valid at this link: https://git-scm.com/docs > Reference

**Comment by Zi-Tao** [comment]
@Vibhu-Kansal 
https://docs.github.com/en/search-github/searching-on-github/searching-issues-and-pull-requests
https://cli.github.com/manual/gh_issue

**Reply by Zi-Tao** [reply]
@Vibhu-Kansal If my answer is helpful, please consider marking me as the answer. Thank you!
